src/index.html

```diff
@@ -220,27 +220,25 @@
     <tl-icon set="tl" name="weaver-w" color="#500000" size="34px" hidden-in-mobile="true"></tl-icon>
     <tl-icon set="bootstrap" name="arrow-right-circle" size="32px" animate="{click: 'rotateToggle'}"></tl-icon>
   </div>
-
   <br /><hr /><br />
   <div class="container" style="height: 700px">
-
-    <tl-mega-menu menu-title="Services"  view-all-href="/view-all" view-all-button-text= "Customized View All Button" hidden-in-mobile="true">
+    <tl-mega-menu menu-title="Services"
+                  view-all-href="/view-all"
+                  view-all-button-text= "Customized View All Button"
+                  hidden-in-mobile="true">
+      <tl-mega-menu-custom top-content>
+          <div class="alert alert-primary" role="alert">
+            <wvr-text value="Primary Alert Message (top-content)Primary Alert Message (top-content)
+              Primary Alert Message (top-content)Primary Alert Message (top-content)Primary Alert Message (top-content)
+              Primary Alert Message (top-content)Primary Alert Message (top-content)Primary Alert Message (top-content)">
+            </wvr-text>
+          </div>
+      </tl-mega-menu-custom>
+      <tl-mega-menu-custom top-content>
```

Comment:
This should be included only as a child of the `tl-mega-menu-custom`

Reply:
Added as child of tl-mega-menu-custom

Reply:
Was intended to added a tl-mega-menu-custom as a sibling element to tl-mega-menu and also as a child element.